code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -867,8 +867,8 @@
 	action_icon_state = "transmit"
 
 /obj/effect/proc_holder/spell/targeted/shadowlingAscendantTransmit/cast(list/targets, mob/user = usr)
-	for(var/mob/living/target in targets)
+	for(var/mob/living/simple_animal/ascendant_shadowling/target in targets)
```

Comment:
My understanding was that this is the target of the spell, which is a self cast. The actual announcement stuff is in a different file and it's borrowed from announce.dm